.github/workflows/run_job.yaml

```diff
@@ -0,0 +1,41 @@
+name: Run data collection
+
+on:
+  push:
+    paths:
+      - '.tracked_repos'
+      - '**.R'
+      - '.github/workflows/run_job.yaml'
+  schedule:
+    - cron:  '59 23 * * 0'
+
+jobs:
+  data-collection:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: r-lib/actions/setup-r@master
+
+      - name: Set GIT
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "Github Action"
+
+      - name: Run Rscript
+        run: Rscript update.R
+
+      - name: Get Date
+        id: date
+        run: echo "::set-output name=date::$(TZ=UTC date +%Y-%m-%d_%H-%M-%S)"
+
+      - name: Commit new data
+        run: |
+          git add --all
+          git commit -m "New data collected at ${{ steps.date.outputs.date }}"
+
+
+      - name: Push data
+        uses: ad-m/github-push-action@master
```

Comment:
is there a way to make this push-to-current-branch?

I saw this but didn't know how to apply here

https://stackoverflow.com/a/58034787/3576984

Reply:
It is a little bit more complicated but it can be done. Small PR solving this coming your way.